.github/test-spec.yml

```diff
@@ -34,8 +34,18 @@ CI-homekit-test:
   - 'mpsl/lib/**/*'
 
 CI-thread-test:
+  - 'softdevice_controller/include/**/*'
+  - 'softdevice_controller/lib/**/*'
   - 'mpsl/include/**/*'
   - 'mpsl/lib/**/*'
+  - 'nrf_802154/driver/**/*'
+  - 'nrf_802154/serialization/**/*'
+  - 'nrf_802154/sl/**/*'
+  - 'crypto/**/*'
```

Comment:
```
    - '!crypto/doc/**/*'
    - '!crypto/*.rst'
    - 'nrf_security/**/*'
    - '!nrf_security/*.rst'
    - '!nrf_security/doc/**/*'
```

Reply:
Can you do the changes directly on the branch?
I'm not sure I understand what you like to be changed.